### PR TITLE
Add parent header hash to log

### PR DESCRIPTION
**Pull request by drahnr**
skip check-dependent-cumulus

### CODE REVIEW

**Comment by drahnr** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4419** 
    * **PR #4421**  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh) and will continue to be automatically updated while this PR remains open.

**Review by ordian — APPROVED** [review]

**Comment by ordian** [comment]
needs a rebase/stack fix

File: runtime/parachains/src/paras_inherent/mod.rs [hunk 1]
```diff
@@ -261,9 +261,12 @@ impl<T: Config> Pallet<T> {
 			mut disputes,
 		} = data;
 
+		let parent_header_hash = parent_header.hash();
+
 		log::debug!(
 			target: LOG_TARGET,
-			"[enter] bitfields.len(): {}, backed_candidates.len(): {}, disputes.len() {}",
+			"[enter_inner] parent_header={:?} bitfields.len(): {}, backed_candidates.len(): {}, disputes.len() {}",
```
Comment (pepyakin) [inline runtime/parachains/src/paras_inherent/mod.rs:268]:
```suggestion
			"[enter_inner] parent_header={:?} bitfields.len(): {}, backed_candidates.len(): {}, disputes.len(): {}",
```

**Review by pepyakin — APPROVED** [review]

**Comment by drahnr** [comment]
bot merge

**Comment by drahnr** [comment]
bot merge

**Comment by drahnr** [comment]
skip check-dependent-cumulus

**Comment by drahnr** [comment]
bot merge

**Comment by ordian** [comment]
bot merge

**Pull request merged**